x-pack/test/api_integration/deployment_agnostic/apis/observability/apm/constants/archiver.ts

```diff
@@ -7,24 +7,20 @@
 
 type ArchiveName =
   | '8.0.0'
-  | 'apm_8.0.0'
   | 'apm_mappings_only_8.0.0'
   | 'infra_metrics_and_apm'
   | 'metrics_8.0.0'
-  | 'ml_8.0.0'
   | 'observability_overview'
   | 'rum_8.0.0'
   | 'rum_test_data';
 
 export const ARCHIVER_ROUTES: { [key in ArchiveName]: string } = {
   '8.0.0': 'x-pack/test/apm_api_integration/common/fixtures/es_archiver/8.0.0',
-  'apm_8.0.0': 'x-pack/test/apm_api_integration/common/fixtures/es_archiver/apm_8.0.0',
   'apm_mappings_only_8.0.0':
     'x-pack/test/apm_api_integration/common/fixtures/es_archiver/apm_mappings_only_8.0.0',
   infra_metrics_and_apm:
     'x-pack/test/apm_api_integration/common/fixtures/es_archiver/infra_metrics_and_apm',
   'metrics_8.0.0': 'x-pack/test/apm_api_integration/common/fixtures/es_archiver/metrics_8.0.0',
-  'ml_8.0.0': 'x-pack/test/apm_api_integration/common/fixtures/es_archiver/ml_8.0.0',
```

Comment:
not supported on serverless

Reply:
In the future, we could create our deployment-agnostic archives to only have supported features/configurations.

Reply:
We should actually go for synthtrace instead. Archives are prone to these kinda of problems.